content/ko/docs/faq/_index.md

```diff
@@ -168,70 +169,83 @@ containers:
-referred with `helm list` by changing the current namespace context (e.g. `helm
-list --namespace foo`).
+헬름 3에서는 특정 릴리스에 대한 정보가 릴리스 자체와 동일한 네임스페이스에
+저장됩니다. 즉, 이제 사용자는 별도의 두 네임스페이스에서
+`helm install wordpress stable/wordpress`를 수행할 수 있으며,
+현재 네임스페이스 컨텍스트를 변경하여 `helm list` 를 조회할 수 있습니다.
+(예: `helm list --namespace foo`)
 
-With this greater alignment to native cluster namespaces, the `helm list` command
-no longer lists all releases by default. Instead, it will list only the releases
-in the namespace of your current kubernetes context (i.e. the namespace shown
-when you run `kubectl config view --minify`). It also means you must supply the
-`--all-namespaces` flag to `helm list` to get behaviour similar to Helm 2.
+이렇게 네이티브 클러스터 네임스페이스가 보다 개선되면서
+`helm list` 명령은 기본적으로 모든 릴리스를 나열하지 않습니다.
+대신 현재 쿠버네티스 컨텍스트의 네임스페이스에 있는 릴리스만
+나열됩니다(즉, `kubectel config view --minify`를 실행할 때 표시되는 네임스페이스).
+또한 헬름 2와 유사한 동작을 수행하려면 `helm list`에
+`--all-namespaces` 플래그를 주어야 합니다.
 
-### 시크릿(secret)이 기본 스토리지 드라이버로
+### 기본 스토리지 드라이버로서의 시크릿(Secret)
 
-In Helm 3, Secrets are now used as the [default storage driver](/docs/topics/advanced/#storage-backends). Helm 2 used ConfigMaps by default to store release information. In Helm 2.7.0, a new storage backend that uses Secrets for storing release information was implemented, and it is now the default starting in Helm 3.
+헬름 3에서 시크릿은 이제 [기본 스토리지
+드라이버](/docs/topics/advanced/#storage-backends)로 사용됩니다. 헬름 2는
+기본적으로 컨피그맵(ConfigMap)을 사용하여 릴리스 정보를 저장합니다. 헬름 2.7.0에서
+릴리스 정보를 저장하기 위해 시크릿을 사용하는 새로운 스토리지 백엔드가 구현되었으며
+헬름 3부터는 기본값이 되었습니다.
 
-Changing to Secrets as the Helm 3 default allows for additional security in protecting charts in conjunction with the release of Secret encryption in Kubernetes.
+헬름 3 기본값을 시크릿으로 변경하면 쿠버네티스의
+시크릿 암호화(Secret Encryption) 릴리스와 함께
+차트를 보호할 수 있습니다.
 
-[Encrypting secrets at rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/) became available as an alpha feature in Kubernetes 1.7 and became stable as of Kubernetes 1.13. This allows users to encrypt Helm release metadata at rest, and so it is a good starting point that can be expanded later into using something like Vault.
+[미사용 시크릿을 암호화하는
+것](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/)은
+쿠버네티스 1.7에서 알파 기능으로 사용 가능하게 되었고
+쿠버네티스 1.13을 기점으로 안정화되었습니다. 이를 통해 사용자는
+미사용 헬름 릴리스 메타데이터를 암호화할 수 있으므로
+나중에 볼트(Vault)와 같은 용도로 확장할 수 있습니다.
 
-### Go 임포트 경로 변경
+### Go import 경로 변경
 
-In Helm 3, Helm switched the Go import path over from `k8s.io/helm` to
-`helm.sh/helm/v3`. If you intend to upgrade to the Helm 3 Go client libraries,
-make sure to change your import paths.
+헬름 3에서 헬름은 Go import 경로를 `k8s.io/helm`에서
+`helm.sh/helm/v3`으로 전환했습니다. 헬름 3 Go 클라이언트 라이브러리로
+업그레이드하려면 import 경로를 변경해야 합니다.
 
-### 수용능력
+### Capabilities
 
-스테이지 렌더링시에 사용가능한 `.Capabilities` 빌트인 객체가 간소화되었다.
+렌더링 단계에서 사용 가능한 `.Capabilities` 빌트인 객체가
+간소화되었습니다.
 
-[빌트인 객체](/docs/chart_template_guide/builtin_objects/)
+[빌트인 객체](/ko/docs/chart_template_guide/builtin_objects/)

```

Comment:
You do not need to specify this - if it exists, and the view is set to `ko`, it is loaded automatically.

Reply:
Should I remove all the other "/ko" strings? (L47, L238)

Reply:
In general you can scroll down to the Checks in the pull request and look at the helm-merge preview: https://deploy-preview-841--helm-merge.netlify.app/ in this case. You can choose Korean from the menu. And then you can test and see if it displays as desired without hardcoding `ko` into your URLs.

Reply:
> In general you can scroll down to the Checks in the pull request and look at the helm-merge preview: https://deploy-preview-841--helm-merge.netlify.app/ in this case. You can choose Korean from the menu. And then you can test and see if it displays as desired without hardcoding `ko` into your URLs.

It does not load automatically. [link](https://deploy-preview-841--helm-merge.netlify.app/ko/docs/faq/#capabilities)

Reply:
Okay; I'll merge and check.

Reply:
(After the other corrections are made. Thanks!)